include/Conversion/PolyToStandard/PolyToStandard.td

```diff
@@ -0,0 +1,21 @@
+#ifndef INCLUDE_CONVERSION_POLYTOSTANDARD_POLYTOSTANDARD_TD_
+#define INCLUDE_CONVERSION_POLYTOSTANDARD_POLYTOSTANDARD_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def PolyToStandard : Pass<"lower-poly"> {
+  let summary = "Lower `poly` to standard MLIR dialects.";
+
+  let description = [{
+    This pass lowers the `poly` dialect to standard MLIR, a mixture of affine,
+    tensor, and arith.
+  }];
+  let dependentDialects = [
+    "mlir::arith::ArithDialect",
+    "mlir::heir::poly::PolyDialect",
+    "mlir::linalg::LinalgDialect",
```

Comment:
do we expect linalg to surface in the result? is dependent dialects necessary dependents or signaling the dialects that will be surfaced from the pass?

Reply:
Removed the non-arith dialects from this list just in case we don't end up using them